crates/router/Cargo.toml

```
@@ -94,6 +93,9 @@ aws-sdk-s3 = "0.25.0"
 aws-config = "0.55.1"
 infer = "0.13.0"
 
+[target.'cfg(not(target_os = "windows"))'.dependencies]
+signal-hook-tokio = { version = "0.3.1", features = ["futures-v0_3"], optional = true }
+
```

seems like some of the CI checks are failing because of windows target `cfg` that we have added in the `router/Cargo.toml`. Let's fix that and get this PR all set for merge shall we.

```suggestion
[target.'cfg(not(target_os = "windows"))'.dependencies]
signal-hook-tokio = { version = "0.3.1", features = ["futures-v0_3"] }

```